src/notes/KeywordStructuredFieldPlugin.jsx

```diff
@@ -29,6 +29,7 @@ function KeywordStructuredFieldPlugin(opts) {
 
     function replaceAllRelevantKeywordsInBlock(curNode, curTransform, state) {
         const listOfKeywordShortcutMappings = getKeyToActiveKeywordShortcutMappings();
+        // console.log('listOfKeywordShortcutMappings: ', listOfKeywordShortcutMappings);
```

Comment:
is there a reason this is still here?

Reply:
Nope. Probably slipped when I rebased. Just removed it